alerter/Alerter.java

```diff
@@ -0,0 +1,64 @@
+class Alerter {
+    static int alertFailureCount = 0;
+
+    static final int THRESHOLD_VALUE = 200;
+
+    static final String ENV_TYPE = "DEVELOPMENT";

```

Comment:
Writing code with environment type switches is not a good idea. The production code should not be aware of where it is executing i.e. test envt or development or production. If behavior needs to be altered then interfaces need to be defined and implementation of these interfaces can be test or production specific.